packages/core/src/sync-store/encoding.ts

```diff
@@ -372,10 +372,8 @@ export const encodeTrace = ({
   type: trace.trace.type,
   gas: hexToBigInt(trace.trace.gas),
   gas_used: hexToBigInt(trace.trace.gasUsed),
-  error: trace.trace.error ? trace.trace.error.replace(/\0/g, "") : null,
-  revert_reason: trace.trace.revertReason
-    ? trace.trace.revertReason.replace(/\0/g, "")
-    : null,
+  error: trace.trace.error ?? null,
+  revert_reason: trace.trace.revertReason ?? null,
```

Comment:
Hmm, I think we still want these? These aren't passing through the Drizzle text column, right?